spec/requests/social_previews_spec.rb

```diff
@@ -6,31 +6,70 @@
   let(:organization) { create(:organization) }
   let(:article) { create(:article, user_id: user.id) }
 
+  before do
+    stub_request(:post, /hcti.io/).
+      to_return(status: 200,
+                body: '{ "url": "https://hcti.io/v1/image/6c52de9d-4d37-4008-80f8-67155589e1a1" }',
```

Comment:
What do you think about extracting the return-url into a `let` that you can re-use here and the assertions